src/handlers.rs

```diff
@@ -118,6 +118,26 @@ impl Handler {
             registration_manager: None,
         })
     }
+
+    pub async fn run(
+        mut self,
+        mut connection: mpsc::Receiver<WebSocket>,
+    ) -> Result<(), ApplicationError> {
+        // TODO: Do we want to allow a new incoming connection to replace the current one?
```

Comment:
Do we want new incoming connections to replace the current one, wait for the first one to close or be rejected?

Reply:
For development it's convenient to have 2 connections at the same time, but it's not necessary. In the registration process, the websocket connection is lost, because we need to redirect to the captcha page. There we need to replace it immediately.